paparazzi-annotations/src/main/java/app/cash/paparazzi/annotation/api/Paparazzi.kt

```diff
@@ -0,0 +1,5 @@
+package app.cash.paparazzi.annotation.api
+
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.BINARY)
+annotation class Paparazzi
```

Comment:
Are you sure about the name? It seems a bit too generic to me, Potentially lending itself to being a god-annotation with many disparate parameters in the future.

Reply:
yep, we don't expect to use this outside of `@Preview` nor to add any parameters.  also, fine to break in the future if we're wrong.